llvm/test/CodeGen/SBF/atomics_sbf.ll

```diff
@@ -181,8 +181,14 @@ entry:
 
 ; CHECK-LABEL: test_min_32
 ; CHECK: ldxw w0, [r1 + 0]
-; CHECK: mov32 w3, w0
-; CHECK: jslt w0, w2,
+; CHECK: mov64 r4, r0
+; CHECK: lsh64 r4, 32
+; CHECK: arsh64 r4, 32
+; CHECK: mov32 r5, w2
+; CHECK: lsh64 r5, 32
+; CHECK: arsh64 r5, 32
+; CHECK:mov32 w3, w0
```

Comment:
shouldn't here be a space between `CHECK:` and `mov32`?

Reply:
That slipped through.